bin/add_practice_exercise

```diff
@@ -30,51 +37,60 @@ command -v curl >/dev/null 2>&1 || {
     exit 1
 }
 
+# Check if configlet is already fetched
+if [ -e bin/configlet ]; then
+    message "success" "Configlet found!"
+else
+    message "info" "Fetching configlet"
+    bin/fetch-configlet
+    message "success" "Fetched configlet successfully!"
+fi
```

Comment:
I would always fetch configlet, as that way you're guaranteed to work with the latest version.